### ISSUE TITLE
Circle of Fifths Inputs

### ISSUE THREAD

**Issue by DaubRidArt** [knchaffin/Meander#6]
The Gate and Position Inputs in the Circle of Fifths graphic are not showing.
Running VCV 1.0.6 on iMac 27 with OSX 10.13.6

**Comment by knchaffin** [comment]
Hi.  Could you please confirm which version of the PurrSoftware Meander plugin you are running and where and when you obtained it?  I.E., did you download it from the VCV Library, did you build it yourself from GitHub, or did you get it from one of the Azure builds?  The reason I ask is that the position and gate inputs were added fairly recently.  If those are the only two things not showing, you may have an older version of the plugin.  Thanks.

**Comment by DaubRidArt** [comment]
Good Day Ken

Thanks for getting back to me.
I hope I am being of some use to you.

Version 1.0.2 loaded from the plugin manager.

See attached screen grab.

**Comment by knchaffin** [comment]
I cannot see the screen grab.  I just loaded Meander from the plugin library, v1.0.2 on Windows and I see the two ports inside of the circle of fifths.  I did rename the "Position" port to "Degree".  Do you see the port jacks at all?  Or is it just the ports label text that is missing.  At the moment I'm baffled as how they could not show up on MacOS.  Do any other ports appear to be missing?

**Comment by DaubRidArt** [comment]
The port that is almost in the centre is labeled ‘Degree’
There is no ‘In’ port below it although the word ‘Gate’ is where it should be.

It looks like your previous version.
However it shows ver 1.0.2 in the cntrl click drop down.

I can see how it is meant to render by checking your manual (one red port, one blue port)

Can I send a different image format screen grab?

**Comment by knchaffin** [comment]
The Degree port is the same as the Position port.  I forgot to change the screen capture image showing in my manual.  If the rest of the knobs and buttons appear to have CV control input jacks to the left of them, per the manual image, the problem is probably on my end.  I will have to dig into it to see if I can figure out what is happening.  V1.0.2 moved to a complete runtime generation of the panel graphics, including the positioning of all jacks.  I'm suspecting that this one jack is being misplaced, much like the one you should see up at the top of the panel between the harmony and melody playing notes display.  That was related to a removed input jack and I fixed it yesterday for V1.0,3 .  Hopefully I can find the problem.  I'll keep you updated.

**Comment by knchaffin** [comment]
I found the bug.  For that input jack, I had a code error that resulted in the jack being positioned outside of the panel height, below the panel where you cannot see it.  This bug was such that at different times and possibly under different compilers, it might work  or not work do to how two variables with the same name but different scope are handled.  This will have to wait on the V1.0.3 release which I am working on now.  V1.0.3 will look quite different inside of the circle of 5ths since I added a STEP button and CV control jack for manually stepping the progression or controlling  via an input pulse from a sequencer, etc.  The entire panel jack positions have been adjusted in V1.0.3 to align everything better and more consistently.  I estimate that I should submit V1.0.3 to the library within a week.  Actually, I will finish up what I am adding now (clock output) and do the release as soon as I finish and do testing.

Thanks for bringing this bug to my attention.
